src/NetAppFiles/NetAppFiles/Common/AzureNetAppFilesCmdletBase.cs

```diff
@@ -52,7 +48,7 @@ public IAzureNetAppFilesManagementClient AzureNetAppFilesManagementClient
             }
             set { _netAppFilesManagementClient = value; }
         }
-
+        /*
```

Comment:
remove this code rather than commenting it out

Reply:
done